internal/worker/stateconverter/interfaces.go

```diff
@@ -21,6 +20,6 @@ type Machiner interface {
 // Machine represents necessary methods for this worker from the
 // a machiner's machine.
 type Machine interface {
-	Jobs() (*params.JobsResult, error)
+	IsController(string) (bool, error)
```

Comment:
I'm just thinking now (a bit late), that this method should be called `IsMachineController()` instead, since we agreed domain service layers are not entities.

Reply:
Yeah I think this is accurate, will fix 👍